rpc/wrtc_call_queue_mongodb.go

```diff
@@ -344,7 +339,7 @@ func (queue *mongoDBWebRTCCallQueue) operatorLivenessLoop() {
 	ticker := time.NewTicker(operatorStateUpdateInterval)
 	defer ticker.Stop()
 	for {
-		if !utils.SelectContextOrWaitChan(queue.cancelCtx, ticker.C) {
+		if !utils.SelectContextOrWaitChan(queue.activeStoppableWorkers.Context(), ticker.C) {
```

Comment:
As per the comment on having `operatorLivenessLoop` accepting a `ctx`, this is the line that caught my attention. We should just be checking the input `context` and not need to do this lookup on the stoppable workers itself.